README.adoc

```diff
@@ -1,3 +1,10 @@
 = Leo Rover Docking
```

Comment:
I would change the readme back to .md format since that is the one Leo Rover guys are using generally...